Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,250 @@
+| Maastricht        |
+| Dordrecht         |
+| Leiden            |
+| Haarlemmermeer    |
+| Zoetermeer        |
+| Emmen             |
+| Zwolle            |
+| Ede               |
+| Delft             |
+| Heerlen           |
+| Alkmaar           |
++-------------------+
+28 rows in set (0.00 sec)
+--------------------------------------------------------------
+
+2.What is the population of Rotterdam ?
+
+mysql> SELECT population FROM city WHERE name='Rotterdam';
++------------+
+| population |
++------------+
+|     593321 |
++------------+
+1 row in set (0.05 sec)
+--------------------------------------------------------------
+
+3.What's the name of all the countries on the continent �North America� ?
+
+mysql> SELECT name FROM country WHERE continent='North America';
++----------------------------------+
+| name                             |
++----------------------------------+
+| Aruba                            |
+| Anguilla                         |
+| Netherlands Antilles             |
+| Antigua and Barbuda              |
+| Bahamas                          |
+| Belize                           |
+| Bermuda                          |
+| Barbados                         |
+| Canada                           |
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+--------------------------------------------------------------
+
+4.What's the top 10 countries by SurfaceArea ?
+
+mysql> SELECT name FROM country ORDER BY SurfaceArea DESC LIMIT 10;
++--------------------+
+| name               |
++--------------------+
+| Russian Federation |
+| Antarctica         |
+| Canada             |
+| China              |
+| United States      |
+| Brazil             |
+| Australia          |
+| India              |
+| Argentina          |
+| Kazakstan          |
++--------------------+
+10 rows in set (0.00 sec)
+
+OR..
+mysql> SELECT name FROM country GROUP BY SurfaceArea DESC LIMIT 10;

```

Comment:
😱not even sure whow this works 😅  first query is way more clear and also easier for the db to make (and easier to optimize)1